app/models/activity.rb

```diff
@@ -621,8 +621,8 @@ def ensure_linked_activity_reciprocity
   end
 
   def linkable_activities
-    return [] unless (programme? || project?) && is_ispf_funded?
-    return [] if project? && parent.linked_activity.nil?
+    return [] unless is_ispf_funded?
```

Comment:
[The commit](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1911/commits/de1247c0c6f0bfd5e449e127850d144bf908c2f1) message of the commit with this change is quite similar to the [previous one](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1911/commits/1018f70b911a83438c2f9e33679baf13645a7df3), and the [subsequent one](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1911/commits/0ed1856cdf90f1c297e83fa13dc1e3e8ebef37bf).

Can they be squashed together or reworded to make it a bit clearer what's going on for each?

Reply:
One of the reasons for splitting them off was ease of reviewing, so if you as a reviewer would be ok with them squished together, we can do that!

Reply:
Yeah, let's do it once we're ready to merge